docs/deploy/create_server.rst

```diff
@@ -115,49 +115,60 @@ If you couldn't select Ubuntu above, follow these additional steps:
 
       reboot
 
-2. Apply core changes
+2. Set up DNS records
+---------------------
+
+#. Create a new hostname DNS entry in `GoDaddy <https://dcc.godaddy.com/manage/OPEN-CONTRACTING.ORG/dns>`__
+
+All server hostnames should be in the following format ocpXX.open-contracting.org (e.g. ocp01, ocp02 etc).
+Each server should have a unique hostname so for each new server, increment the counter by one. 
+To see which hostnames are currently in use `refer to the salt roster config <https://github.com/open-contracting/deploy/blob/master/salt-config/roster>__`.
+
+
+3. Apply core changes
 ---------------------
 
 #. Connect to the server as the ``root`` user using SSH, and change its password, using the ``passwd`` command. Use a `strong password <https://www.lastpass.com/password-generator>`__, and save it to OCP's `LastPass <https://www.lastpass.com>`__ account.
 
-   .. note::
+.. note::
 
-      The root password is needed if you can't login via SSH (for example, due to a broken configuration). For Bytemark, open the `panel <https://panel.bytemark.co.uk/servers>`__, click the server's *Console* button, and login.
+   The root password is needed if you can't login via SSH (for example, due to a broken configuration). For Bytemark, open the `panel <https://panel.bytemark.co.uk/servers>`__, click the server's *Console* button, and login.
```

Comment:
I think removing the indentation here causes the numbering to re-start after the note.

Reply:
Fixed